examples/adc.rs

```diff
@@ -3,10 +3,11 @@
 use core::fmt::Write;
 use libtock::adc;
 use libtock::console::Console;
-use libtock::result::TockResult;
 use libtock::timer;
 use libtock::timer::Duration;
+use libtock_support_macros::libtock_main;
```

Comment:
The macro should be reexported, s.t. you do not need to specify the dependency on `libtock_support_macros`.

Reply:
done